jobserver/models.py

```diff
@@ -369,6 +369,9 @@ class Project(models.Model):
     proposed_start_date = models.DateTimeField(null=True, blank=True)
     proposed_duration = models.TextField(blank=True)
 
+    # store a link to our modified FormA
+    form_url = models.TextField(blank=True)
```

Comment:
Worth calling this `form_a_url`?

Reply:
Not in this instance I think.  We currently ship a GDoc with a form to potential project registrations (modified from NHS E's Form A) but it's both not Form A and expected to be pulled into the app in the next iteration.